core/src/main/java/org/apache/iceberg/CachingCatalog.java

```diff
@@ -36,6 +35,7 @@
 import org.apache.iceberg.exceptions.AlreadyExistsException;
 import org.apache.iceberg.relocated.com.google.common.base.Preconditions;
 import org.apache.iceberg.relocated.com.google.common.collect.ImmutableList;
+import org.checkerframework.checker.nullness.qual.NonNull;
```

Comment:
Can you remove this? I we don't typically use these annotations.

Reply:
sure :smile:

Reply:
done